tests/template/test_kedro_cli.py

```diff
@@ -224,6 +242,9 @@ def test_run_with_config(
             load_versions={},
             pipeline_name="pipeline1",
         )
+        fake_load_context.assert_called_once_with(
+            Path.cwd(), env=None, extra_params=expected
```

Comment:
We don't really care what it was called with `env=None` here, so maybe:

```suggestion
            Path.cwd(), env=mocker.ANY, extra_params=expected
```